spec/suites/map/MapSpec.js

```diff
@@ -1336,4 +1336,57 @@ describe("Map", function () {
 			}).to.not.throwException();
 		});
 	});
+
+	describe("_addZoomLimit", function () {
+		it("update zoom levels when min zoom is not NaN in a layer that is added to map", function () {
```

Comment:
Please check each test describtion, the min zoom is a number. I think copy&paste issue.